site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -86,7 +98,7 @@ Flag explanations:
 * scopes: Defines the Oauth scopes required by the nodes.
 * num-nodes: The number of nodes to be created in each of the cluster's zones. Default: 4. Depending on the needs of your game, this parameter should be adjusted.
 * no-enable-autoupgrade: Disable automatic upgrades for nodes to reduce the likelihood of in-use games being disrupted.
-* machine-type: The type of machine to use for nodes. Default: n1-standard-4. Depending on the needs of your game, you may wish to [have smaller or larger machines](https://cloud.google.com/compute/docs/machine-types).
+* machine-type: The type of machine to use for nodes. Default: e2-medium. Depending on the needs of your game, you may wish to [have smaller or larger machines](https://cloud.google.com/compute/docs/machine-types).
```

Comment:
Curious - why `e2-medium` and not `e2-standard-4` ?

Reply:
This is the machine type that is listed as the default from running `gcloud container clusters create --help`:

```
     --machine-type=MACHINE_TYPE, -m MACHINE_TYPE
        The type of machine to use for nodes. Defaults to e2-medium. The list
        of predefined machine types is available using the following command:

            $ gcloud compute machine-types list
```

Reply:
🤔 I already merged this, but e2-standard-4 is what we actually default things to, e2-medium is ambiguous, no?

Reply:
I guess it's a question of whether we are saying what _our_ default value is in the command above, or the default value from gcloud if you don't specify this argument. I was reading it as the latter. Reading it again, it's probably the former. I'll update the default value in my next PR.